requirements/prod.txt

```diff
@@ -184,24 +191,37 @@ kombu==3.0.35 \
 m2secret==0.1.1 \
     --hash=sha256:4174f4967b378d751e758894d6c394db506c4f8753d1db66494320c31667d8a0
 # mock is required by amo-validator
-mock==1.0.1 \
-    --hash=sha256:b839dd2d9c117c701430c149956918a423a9863b48b09c90e30a6013e7d2f44f \
-    --hash=sha256:8f83080daa249d036cbccfb8ae5cc6ff007b88d6d937521371afabe7b19badbc
+mock==2.0.0 \
+    --hash=sha256:5ce3c71c5545b472da17b72268978914d0252980348636840bd34a00b5cc96c1 \
+    --hash=sha256:b158b6df76edd239b8208d481dc46b6afd45a846b7812ff0ce58971cf5bc8bba
 moz-addon-packager==1.0.1 \
     --hash=sha256:57bafb7c5eea6a91b53542ac485ce94219652c6024ccb8aae8b26ce275e0b0e3
 mozilla-logger==0.2 \
     --hash=sha256:085db73901ea38728ac5eb14197979ae8b9524511ba7f0d8941d70650b586ef5
+# natsort is required by amo-validator
+natsort==5.0.1 \
+    --hash=sha256:ad35be68ef6db20a9b0ae6c81df8ea1c6166f924e92de967b9adca97912acec6 \
+    --hash=sha256:4ad6b4d1153451e345967989bd3ca30abf33f615b116eeadfcc51a456e6974a9 \
+    --hash=sha256:62648ac68cf55193bcb4c1078cf99dfb72bb7ed6c1bacca5fd36b6167952b5c6
 ndg-httpsclient==0.4.1 \
     --hash=sha256:133931ab2cf7118f8fc7ccc29e289ba8f644dd90f84632fa0e6eb16df4ba1891
 newrelic==2.66.0.49 \
     --hash=sha256:f95b90def0c86b6d4f859664ca411bae90da76345f23ed2de40531d39e2b32a1
 nobot==0.4.1 \
     --hash=sha256:e835abfe9b813fe5697475e8fc0dfbaf9c7572f4574e3d2d7e5114bc84f4cf0b \
     --hash=sha256:bae7bc785e81d4edcc7e7e66457697995c47475e0a2aba405b3a5cb82697e880
+# nose is required by amo-validator
+nose==1.3.7 \
```

Comment:
We should convert amo-validator to using py.test to be able to remove this one. It's probably straightforward.